src/services/twilio-help/twilio-command-help.js

```diff
@@ -31,6 +33,30 @@ class TwilioCommandHelp extends CommandHelp.default {
     return returnList.join('\n');
   }
 
+  // overriding to include help document link
+  examples(examples) {
+    if ((examples === undefined || !examples || examples.length === 0) && this.command.docLink === '') return '';
+    const listOfDetails = [];
+    if (examples !== undefined && examples.length !== 0) {
```

Comment:
Isn't this check redundant as we already do it 2 lines above? 

Also, would love to see what the example object would look like to understand this piece of code better

Reply:
The first check is to do a sanity check and make an early return. Since it has 2 components examples and helpDocument so I had to make two separate if blocks to add the details, only if they're present. However, I'll be adding a new func for only doc link, as Anuj suggested. So, this'd be gone. If you want to see how examples originally behave, check [this](https://github.com/oclif/plugin-help/blob/712c40c7e270065ab82dfae3c4ea56aaa91a4e48/src/command.ts#L89), it's essentially a string array with predefined commands (autocomplete command has examples object, for reference).